python/paddle/fluid/tests/unittests/test_uniform_random_op.py

```diff
@@ -26,6 +26,8 @@
 import paddle.fluid as fluid
 from paddle.fluid import Program, program_guard
 
+paddle.enable_static()
```

Comment:
Why need this?

Reply:
useless in `uniform_random xpu` kernel.
this line only for run `test_uniform_random_op.py` using `python` rather than `ctest`.
have been deleted.